### PR TITLE
removed hard-coded e, added import from constant file, updated function to take a number rather than a list.

### PR DESCRIPTION
**Pull request by whuras**

### CODE REVIEW

**Comment by whuras** [comment]
disregard PR

**Comment by whuras** [comment]
reopening PR

File: Functions/Cosh.py [hunk 1]
```diff
@@ -2,13 +2,11 @@
...
 
 
-def cosh(args):
-    e = 2.718281828459045 # replace from constant file once created
-    x = 0
-    if(len(args) == 1):
-        x = args[0]
-    else:
-        raise Exception(f"Invalid number of arguments, power_function got {len(args)} but expected 1.")
+def cosh(number):
```
Comment (steveiaco) [inline Functions/Cosh.py]: your input needs to be a list, not a number. i think you accidentally overwrote your previous work

File: Functions/Cosh.py [hunk 2]
```diff
@@ -2,13 +2,11 @@
 # Taylor: cosh(x) = 1 + (x^2/2!) + (x^4/4!) + (x^6/6!) ...
 
 from Power_Function import power_function
+from Constants import get_e()
```
Comment (steveiaco) [inline Functions/Cosh.py]: this should be get_e, not get_e(), otherwise it doesn't run

**Review by steveiaco — APPROVED** [review]
lgtm!

**Pull request merged**